docs/DeveloperGuide.md

```diff
@@ -275,15 +254,11 @@ the filtered transaction with a success message.
 - 2a. User enters an incorrect format.
     - 2a1. UniCa$h displays an error message, requests for correct format.
     - 2a2. User enters command with new format.
-    Steps 2a1-2a2 are repeated until the format entered is correct.
+    - Steps 2a1-2a2 are repeated until the format entered is correct. 
+    - Use case resumes from Step 3.
```

Comment:
Should this part simply be `Use case resumes at Step 1`?

Reply:
(following your `reset_unicash` extension)

Reply:
Nice, thanks, I've changed it